test/acceptance/test/add_tests.go

```diff
@@ -330,4 +334,52 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 			verifyWegoAddCommandWithDryRun(appRepoName, WEGO_DEFAULT_NAMESPACE)
 		})
 	})
+
+	// Eventually this test run will include all the remaining un-automated `wego app add` flags.
+	It("SmokeTest - Verify 'wego app add' works when --url flag is specified", func() {
+		var repoAbsolutePath string
+		private := true
+		appRepoName := "wego-test-app-" + RandString(8)
+		url := "ssh://git@github.com/weaveworks-gitops-test/" + appRepoName + ".git"
```

Comment:
Same as above => Should not hardcode the org part, instead should fetch from os.Getenv("GITHUB_ORG") environment variable.